src/python/strelka/tests_configuration/test_scanner_assignment.py

```diff
@@ -56,7 +56,7 @@
     ],
     "test.rar": ["ScanRar"],
     "test.tar": ["ScanTar"],
-    "test.txt": ["ScanUrl"],
+    "test.txt": ["ScanUrl", "ScanTlsh"],
```

Comment:
Any scanner that is assigned by wildcard `*` is left out of test_scanner_assignment since otherwise it would match all the fixtures and make the test hard to maintain. It's fine to omit ScanTlsh from this test, and you'll see it causes the test to fail based on this version of `backend.yaml` when you run with `build --build-arg CONFIG_TESTS=true`. `test_distribute` will more or less make sure wildcard assignments are valid, which you updated already.